valiu-node-rpc/tests/add_member.rs

```diff
@@ -0,0 +1,23 @@
+#![cfg(feature = "_integration-tests")]
+
+use sp_keyring::AccountKeyring;
+use substrate_subxt::{extrinsic::PairSigner, ClientBuilder};
+use valiu_node_rpc::{AddMemberCall, ValiuRuntime};
+
+#[tokio::test]
+async fn add_member() {
```

Comment:
Indeed, `attest` and `transfer` will be next

Reply:
The `providers` group used to be a thing but it doesn't exist anymore. Now that `attest` is the only available group, is there a reason to keep using `pallet_membership`? You know, by definition only one account will be able to insert members and this centralization of power is concerning.

Reply:
How would you go about not using `pallet_membership`? There is a centralization of power indeed but at the beginning would have to be this way as we roll out cautiously step by step in a permissioned way as a private blockchain. At the beginning the only people that can do anything with the VLN are the liquidity providers that are hand picked by us, because they have the power to create USDv which is created initially with the attestation system that requires trusting them to have the assets off-chain that they claim to have. As we progress towards decentralization the providers can be later elected by a collective, say other providers, and later it will be ideal to bridge our chain to others(Polkadot) in a trustless way so anyone can bring external assets to our chain to be used as collateral for the minting of USDv in a fully decentralized permissionless fashion.

Reply:
My statement derived from our previous conversations were, in my understanding, an end-user could also attest but looks like an attestation should pass through a liquidity provider as an intermediate endorser or it is something can be achieved in the future. In any case, the `AddOrigin` key should be stored safely.